Templates/Java/BaseJavaModel.template.tt

```diff
@@ -1089,7 +1089,7 @@ public {1} {2}{3}{4} {{";
                 propertyName.SplitCamelCase(),
                 property.Name,
                 propertyType,
-                property.Name.SanitizePropertyName(property).ToLowerFirstChar(),
+                property.Name.ToLowerFirstChar().SanitizePropertyName(property),
```

Comment:
I think `ToLowerFirstChar()` is a no-op as all properties are camelCase.

Reply:
they should be, until you find out that some properties (under the device entity) are not camelCased and it breaks the generation.

Reply:
interesting. Another example as to why we should have a validation tool for metadata for our assumptions in the code generation.

Reply:
yep, here is my PR which attempts to fix the metadata in case you're interested in knowing which properties. https://microsoftgraph.visualstudio.com/onboarding/_git/AGS-OnboardingAutomationPipeline/pullrequest/5283

Reply:
I wonder if it is a rule to have them in camelCase or me doing a hasty generalization :)